File: Launch HTAWeb.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    const char* verbose_info[] = {
        "Setting up Internet Explorer runtime...",
        "Loading up HTA code...",
        "Exiting C loader script..."
    };

    int verbose_info_length = sizeof(verbose_info) / sizeof(verbose_info[0]);

    for (int i = 0; i < verbose_info_length; i++) {
        printf("%s\n", verbose_info[i]);
    }

    const char* hta_file = "htaweb.hta";

    printf("Launching HTA file...\n");

    if (system(hta_file) != 0) {
        printf("Error launching HTA file.\n");
    }

    return 0;
}

File: Launch HTAWeb.cs
using System;

class Program {
    static void Main() {
        string[] verbose_info = {
            "Setting up Internet Explorer runtime...",
            "Loading up HTA code...",
            "Exiting C# loader script..."
        };

        for (int i = 0; i < verbose_info.Length; i++) {
            Console.WriteLine(verbose_info[i]);
        }

        string hta_file = "htaweb.hta";

        Console.WriteLine("Launching HTA file...");

        System.Diagnostics.Process process = new System.Diagnostics.Process();
        process.StartInfo.FileName = hta_file;
        process.StartInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
        process.Start();
        process.WaitForExit();

        if (process.ExitCode != 0) {
            Console.WriteLine("Error launching HTA file.");
        }
    }
}
